source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
```

Comment:
Could "Clone the Project" and "Manual Creation" be in tabs? I think right now the headings go:

- Get NPM
- Download Project
- Manual Creation

This reads like a three step procedure but it is really a two step procedure with a branch in the second step.

Reply:
Agreed -- will tab these two options